django_netjsonconfig/base/admin.py

```diff
@@ -200,6 +205,15 @@ def download_view(self, request, pk):
         return send_file(filename='{0}.tar.gz'.format(config.name),
                          contents=config_archive.getvalue())
 
+    def device_context_view(self, request, pk):
+        device = get_object_or_404(self.model, pk=pk)
+        if hasattr(device, 'config'):
+            device_config = device.config
+        else:
+            device_config = Config(device=device)
```

Comment:
this logic has to be moved into a `get_context` method on `AbstractDevice`, with the difference that in order to get the `Config` model, you will have to use the `get_config_model` method.

Reply:
the `get_context` method on the device then will have to simply return `config.get_context()`

Reply:
please also write a unit test for this new method on the device model in `test_models`

Reply:
I just found out that `AbstractTemplate` class doesn't have `get_context` function, I should make function there too, right?